test/unit/annotations/deprecated.spec.ts

```diff
@@ -60,6 +60,14 @@ test("luaiterator removed", () => {
     `.expectDiagnosticsToMatchSnapshot([annotationRemoved.code]);
 });
 
+test("tuplereturn removed", () => {
+    util.testFunction`
+        /** @tupleReturn */
+        function tuple(): [number, number, number] { return [3, 5, 1]; }
+        return tuple()[2];
+    `.expectDiagnosticsToMatchSnapshot([annotationRemoved.code]);
```

Comment:
Add a test for just:

```
        /** @tupleReturn */
         function tuple(): [number, number, number] { return [3, 5, 1]; };
```
(without the call). This should also result in a diagnostic